clang/include/clang-c/CAS.h

```diff
@@ -185,6 +185,35 @@ CINDEX_LINKAGE void clang_experimental_cas_loadObjectByString_async(
     void (*Callback)(void *Ctx, CXCASObject, CXError),
     CXCASCancellationToken *OutToken);
 
+/**
+ * Stores an object in the CAS.
+ *
+ * \param Refs The refs of the object to be stored.
```

Comment:
I think the Refs are `CASID` here? That is probably a bit confusing since `ObjectRef` is something slightly different.

I am ok with using CASID on API level as it is easier to print but it is not efficient to use this API to write large amount of connected CAS objects (while ObjectRef is very efficient).